### PR TITLE
homework 4. I have done. Check it, please)

### PR DESCRIPTION
**Pull request by borysovdenys**

### CODE REVIEW

File: Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/controller/LoginController.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        req.getRequestDispatcher("index.jsp").forward(req, resp);
+    }
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        LOGGER.info("Log in");
+
+        String login = request.getParameter("login_field");
+        String password = request.getParameter("password_field");
+
+        User currentUser = null;
```
Comment (dinamio) [inline Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/controller/LoginController.java:40]: You can move currentUser inside if, I believe

File: Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/service/UserService.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        User user1 = new User("den", "11", "mail");
+        User user2 = new User("alex", "alex1", "email");
+        users.add(user1);
+        users.add(user2);
+
+    }
+
+    public User getUserByLoginAndPass(String login, String password) throws DAOException {
+        for (User user : users) {
+            System.out.println(user);
```
Comment (dinamio) [inline Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/service/UserService.java:26]: YOu're using loggers, why do you still need sout?)

**Review by dinamio — APPROVED** [review]
Looks good!

File: Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/service/UserService.java (hunk 2 above)
Comment (borysovdenys) [reply Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/service/UserService.java:26]: It was an accident

File: Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/controller/LoginController.java (hunk 1 above)
Comment (borysovdenys) [reply Borysov/homework 4. Session, Filters/Quiz/src/main/java/borysov/controller/LoginController.java:40]: instead of try-catch?

**Pull request merged**